celery/extractions.py

```diff
@@ -117,11 +130,26 @@ def export_schema_to_sql(year, proj, output_dir, conn, pg_connect_string):
             remove(table_output_file)
 
 
+def sendmail(to, message):
+    msg = MIMEText(message.encode('utf-8'), _charset='utf-8')
+    msg['Subject'] = MAIL_SUBJECT
+    msg['From'] = MAIL_FROM
+    msg['To'] = to
+    try:
+        smtpObj = SMTP(SMTP_HOST, SMTP_PORT)
+        smtpObj.sendmail(MAIL_FROM, [to], msg.as_string())
+        smtpObj.quit()
+        print('Successfully sent email to %s' % to)
+    except SMTPException:
+        print('Error: unable to send email to %s' % to)
+
+
 @taskmanager.task(name='extraction.do')
 def do(year, format, proj, email, cities):
 
-    extraction_id = 'foncier_{0}{1}{2}_{3}'.format(year, format, proj, do.request.id)
-
+    uuid = do.request.id
+    extraction_id = 'foncier_{0}_{1}_{2}_{3}'.format(year, format, proj, uuid)
```

Comment:
Please use fixed format : `extraction_id = 'foncier_{0}{1}{2}_{3}'.format(year, format, proj, do.request.id)` with no spaces between `{0}`, `{1}` and `{2}` ?